lxd/storage/drivers/driver_zfs_utils.go

```diff
@@ -252,6 +252,29 @@ func (d *zfs) getDatasetProperty(dataset string, key string) (string, error) {
 	return strings.TrimSpace(output), nil
 }
 
+func (d *zfs) getDatasetProperies(dataset string, keys []string) (map[string]string, error) {
```

Comment:
Lets make keys variadic so we can easily pass in multiple hard code values or from a slice.